epcis-context-protected.jsonld

```diff
@@ -4,11 +4,11 @@
       "@vocab": "https://ns.gs1.org/epcis/",
       "@protected": true,
       "epcis": "https://ns.gs1.org/epcis/",
-      "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
-      "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
-      "owl": "http://www.w3.org/2002/07/owl#",
-      "xsd": "http://www.w3.org/2001/XMLSchema#",
-      "dcterms": "http://purl.org/dc/terms/",
+      "rdf": "https://www.w3.org/1999/02/22-rdf-syntax-ns#",
```

Comment:
@jmcanterafonseca-iota You can't change semantic URLs like this!!!!
If you change even 1 char of a URL, it's a different URL and these standard terms won't be recognized by any semantic repository or other processing system.

Reply:
@jmcanterafonseca-iota  As @VladimirAlexiev  correctly states, it would be wrong to change these CURIE prefixes to https since semantic applications rely on character-by-character exact string matches for URIs.